charts/datagovuk/templates/_find.tpl

```diff
@@ -10,4 +10,8 @@
   value: "1"
 - name: RAILS_DEVELOPMENT_HOSTS
   value: {{ .Values.find.ingress.host }}
+- name: RAILS_SERVE_STATIC_FILES
+  value: "1"
+- name: SECRET_KEY_BASE
+  value: "test00001111222233334444"
```

Comment:
Optional: seems a little surprising to see a dummy value for an env var buried in a partial template like this — can imagine some poor soul (perhaps myself!) spending a while tracking this down while debugging something.

Not a showstopper — fine to merge and improve on the way to production.

Reply:
yes, this is just temporary, I'll probably just copy the secrets across eventually for each environment from PaaS